### DISCUSSION TITLE
ent default connection pool

### DISCUSSION THREAD

**Discussion by wujiang41105631** [ent/ent#3303] [Q&A] [answered]
Does ent use connection pool by default? And it using the Go-native connection pool?

**Comment by a8m** [comment, marked as answer]
Hey @wujiang41105631 👋 
By default, Ent uses `sql.DB` which is a pool of underlying connections (`sql.Conn`). The pool can be controlled using the following API: https://entgo.io/docs/sql-integration.

**Reply by wujiang41105631** [reply]
Thank you ~
